Blish HUD/GameServices/Gw2MumbleService.cs

```diff
@@ -73,6 +88,8 @@ public class Gw2MumbleService : GameService {
         }
 
         protected override void Update(GameTime gameTime) {
+            HandleEvents();
```

Comment:
Move this to after the other update calls in this method are made.  That way when the event is called, the fresh values are already available.